src/evaluator/index.js

```diff
@@ -87,6 +96,11 @@ class Evaluator {
       const inputs = await this.evaluateNodes(node.inputs)
       const outputs = node.outputs
 
+      if (target.type !== 'bytes20' &&
+        target.type !== 'address') {
+        this.panic('Target of call expression was not an address')
+      }
+
```

Comment:
Tiny detail, we should fail if the address is has any uppercase letters but checksum it is incorrect.